style/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: url("https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg") 50% 100% no-repeat;
+  height: 68vh;
+}
+
+footer a,
+footer ul a {
+  text-decoration: none;
+  color: #09c;
+}
+
+header ul a {
+  text-decoration: none;
+}
+
+.nav-menu ul {
+  display: flex;
+  height: 2.5rem;
+  width: 80vw;
+  margin: 5.2rem auto 0 auto;
+  border-radius: 0.3rem;
+  text-align: center;
+  background-image: linear-gradient(#808080 50%, #5e5e5e 30%, #808080 90%);
+}
+
```

Comment:
- [ ] The navbar is too wide. Check the original page to adjust yours and make it look like a clone.

- [ ] You shouldn't have that much space after the search bar. This can improve when you change the width.

![Screenshot from 2020-03-16 23-34-03](https://user-images.githubusercontent.com/51926937/76825151-b547be80-67de-11ea-8f89-3655f27b22c7.png)